raijin_scripts/deploy/dea-env/environment.yaml

```diff
@@ -40,7 +40,6 @@ dependencies:
 - gettext
 - giflib
 - glib
-- glueviz
```

Comment:
Why has this been moved into `reinstall_miniconda.sh`?

Reply:
`glueviz` conda package did not install correctly when it was included within `environment.yaml` file. Need to investigate more on this issue. 
As a work-around, `glueviz` package was moved into `reinstall_miniconda.sh` file.